tests/DCPS/Compiler/XtypesExensibility/run_test.pl

```diff
@@ -0,0 +1,22 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+     & eval 'exec perl -S $0 $argv:q'
+     if 0;
+
+use Env (DDS_ROOT);
+use lib "$DDS_ROOT/bin";
+use Env (ACE_ROOT);
+use lib "$ACE_ROOT/bin";
+use PerlDDS::Run_Test;
+use strict;
+
+my $name = 'Compiler_XtypesExtensibility';
+my $status = 0;
+my $TESTDVR = PerlDDS::create_process ($name);
+
+my $status = $TESTDVR->SpawnWaitKill (300);
```

Comment:
300 seconds, when there would be a problem the build waits 5 minutes before killing the process, is that really required to be so long?